src/test/java/StringClassTest.java

```diff
@@ -0,0 +1,43 @@
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import static org.assertj.core.api.Assertions.*;
+
+public class StringClassTest {
+
+    @Test
+    public void split_V1() {
```

Comment:
```suggestion
    @Test
    @DisplayName("1,2을 ,로 split 했을 때 1과 2로 잘 분리되는지")
    public void split_V1() {
```

- 테스트하고자 하는 대상을 문장으로 명확히 표현하는게 좋습니다. 추후에 어떤 코드인지 파악하는 비용을 줄이고, 테스트코드를 문서화하는 효과가 있어서 타인이 코드를 볼 때도 소통 비용을 줄일 수 있어요.
- 기능 요구사항을 그대로 DisplayName으로 적는 습관을 들이면, 어떤 테스트 케이스를 작성해야하는지 고민하는 비용을 줄일 수 있어요.